vertical-pod-autoscaler/enhancements/5030-vpa-manager/README.md

```diff
@@ -0,0 +1,338 @@
+
+## Proposal
+
+<!--
+This is where we get down to the specifics of what the proposal actually is.
+This should have enough detail that reviewers can understand exactly what
+you're proposing, but should not include things like API designs or
+implementation. What is the desired outcome and how do we measure success?.
+The "Design Details" section below is for the real
+nitty-gritty.
+-->
+
+### User Stories (Optional)
+
+<!--
+Detail the things that people will be able to do if this KEP is implemented.
+Include as much detail as possible so that people can understand the "how" of
+the system. The goal here is to make this feel real for users without getting
+bogged down.
+-->
+
+#### Story 1
+  when we want to create vpa for all workloads in namespace a, we just label namespace a with autoscaling/vpa=open, then vpa manager create vpa for every workload not annotated with autoscaling/vpa=close in namespace a, and later when workload are created in namespace a, corresponding vpa will be automaticlly created.
+
+  when we want to delete all vpas in namespace a, we just label namespace a with autoscaling/vpa=close, then vpa manager delete vpa for every workload not annotated with autoscaling/vpa=open in namespace a.
+
+#### Story 2
+
+  we can configure vpa manger default pod update mode as auto( or initial). when vpa manager create vpa, it will set vpa's updatepolicy.updateMode to auto (or initial). we can also specify update mode for namespace a by labeling namespace a with vpa/update-mode=initial( or vpa/update-mode=initial), in this case, all vpa created in namespace a has initial update mode except those vpa correspoding workloads annotated vpa/update-mode=auto.
+
+  later when we change update mode of namespace a by labeling it, all the vpa's update mode will be changed correspondingly in namespace a except those tarRef has been annotated by vpa/update-mode=***.
+
+### Notes/Constraints/Caveats (Optional)
+
+<!--
+What are the caveats to the proposal?
+What are some important details that didn't come across above?
+Go in to as much detail as necessary here.
+This might be a good place to talk about core concepts and how they relate.
+-->
+
+### Risks and Mitigations
+
+<!--
+What are the risks of this proposal, and how do we mitigate? Think broadly.
+For example, consider both security and how this will impact the larger
+Kubernetes ecosystem.
+
+How will security be reviewed, and by whom?
+
+How will UX be reviewed, and by whom?
+
+Consider including folks who also work outside the SIG or subproject.
+-->
+
+## Design Details
+
+<!--
+This section should contain enough information that the specifics of your
+change are understandable. This may include API specs (though not always
+required) or even code snippets. If there's any ambiguity about HOW your
+proposal will be implemented, this is the place to discuss them.
+-->
+later i will create a pr
```

Comment:
I think we need more details here.

Is it a separate executable? What happens with unusual label configurations? Why are we using labels as an API? What will the updater do (presumably watch namespaces, watch all controllers, watch VPA objects, do something to ensure VPA match labels on namespaces and controllers)

Reply:
it's a seperate executable